kong/llm/drivers/gemini.lua

```diff
@@ -41,30 +41,32 @@ local function is_response_content(content)
         and content.candidates[1].content.parts[1].text
 end
 
-local function is_response_finished(content)
-  return content
-     and content.candidates
-     and #content.candidates > 0
-     and content.candidates[1].finishReason
-end
-
-local function handle_stream_event(event_t, model_info, route_type)  
+local function handle_stream_event(event_t, model_info, route_type)
   -- discard empty frames, it should either be a random new line, or comment
   if (not event_t.data) or (#event_t.data < 1) then
     return
   end
-  
+
+  if event_t.data == "[DONE]" then
```

Comment:
let's move `"[DONE]"` to a const

Reply:
I have moved this everywhere it's used, and re-tested all providers.